src/SDKs/_metadata/dns_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-03-14 21:04:59 UTC
+2018-09-06 21:12:30 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      58409bca13e129632c26c9d3767f22f922af62dc
+Commit:      5ceb33d2e5c0692dbdbcc907e025c1dfd994f29c
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\muwaqar\AppData\Roaming\npm `-- autorest@2.0.4245  
+Bootstrapper version:    C:\Users\karravi\AppData\Roaming\npm `-- autorest@2.0.4283  
```

Comment:
Please pull latest changes from upstream psSdkJson6 and run msbuild build.proj again